@xen-orchestra/web/src/components/host/HostPifSidePanel.vue

```diff
@@ -0,0 +1,459 @@
+<template>
+  <UiPanel>
+    <template #header>
+      <UiButton
+        v-tooltip="$t('coming-soon')"
+        disabled
+        size="medium"
+        variant="tertiary"
+        accent="brand"
+        :left-icon="faEdit"
+      >
+        {{ $t('edit') }}
+      </UiButton>
+      <UiButton
+        v-tooltip="$t('coming-soon')"
+        disabled
+        size="medium"
+        variant="tertiary"
+        accent="danger"
+        :left-icon="faTrash"
+      >
+        {{ $t('delete') }}
+      </UiButton>
+    </template>
+    <template #default>
+      <!-- PIF -->
+      <UiCard class="card">
+        <UiCardTitle>{{ isBond ? $t('bond') : $t('pif') }}</UiCardTitle>
+        <div class="content">
+          <!-- UUID -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('uuid') }}
+            </template>
+            <template #value>
+              {{ pif.id }}
+            </template>
+            <template #addons>
+              <VtsIcon
+                v-if="pif.management"
+                v-tooltip="$t('management')"
+                accent="info"
+                :icon="faCircle"
+                :overlay-icon="faStar"
+              />
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                :icon="faCopy"
+                size="medium"
+                accent="brand"
+                @click="copy(pif.id)"
+              />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- NETWORK -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('network') }}
+            </template>
+            <template #value>
+              <div class="network">
+                <!-- TODO Remove the span when the link works and the icon is fixed -->
+                <!--
+                <UiComplexIcon size="medium">
+                  <VtsIcon :icon="faNetworkWired" accent="current" />
+                  <VtsIcon accent="success" :icon="faCircle" :overlay-icon="faCheck" />
+                </UiComplexIcon>
+                <a href="">{{ networkNameLabel }}</a>
+                -->
+                <span v-tooltip class="text-ellipsis">{{ network?.name_label }}</span>
```

Comment:
Shouldn't we display at least a `-` if `network` is undefined and we have no `name_label`?